index.md

```diff
@@ -56,6 +56,7 @@ Base Tools
 
 -   [WP-CLI](https://github.com/wp-cli/wp-cli) - The command-line tool for managing WordPress.
 -   [WP-CLI GUI](https://wpcligui.com/) - A GUI for the command line interface for WordPress.
+-   [AnsiPress] (https://github.com/AnsiPress/AnsiPress) - The command-line tool for manage WordPress website on NGINX/PHP7/MariaDB/Google-PageSped stack.
```

Comment:
Typos: "...for **managing** WordPress **websites** on ...-Google-**PageSpeed** stack."

I find the whole wording a little off, though.